build.xml

```diff
@@ -601,17 +600,17 @@
           <dependency groupId="org.slf4j" artifactId="slf4j-api" version="2.0.9"/>
           <dependency groupId="org.slf4j" artifactId="log4j-over-slf4j" version="2.0.9"/>
           <dependency groupId="org.slf4j" artifactId="jcl-over-slf4j" version="2.0.9" />
-          <dependency groupId="ch.qos.logback" artifactId="logback-core" version="1.4.14"/>
-          <dependency groupId="ch.qos.logback" artifactId="logback-classic" version="1.4.14"/>
-          <dependency groupId="com.fasterxml.jackson.core" artifactId="jackson-core" version="2.13.2"/>
-          <dependency groupId="com.fasterxml.jackson.core" artifactId="jackson-databind" version="2.13.2.2"/>
-          <dependency groupId="com.fasterxml.jackson.core" artifactId="jackson-annotations" version="2.13.2"/>
+          <dependency groupId="ch.qos.logback" artifactId="logback-core" version="1.5.16"/>
```

Comment:
have you verified that cndb uses the same versions?

Reply:
The dep tree in CNDB gives you:
```
+- ch.qos.logback:logback-core:jar:1.4.14:compile
+- ch.qos.logback:logback-classic:jar:1.4.14:compile
```
and the [pom.xml](https://github.com/riptano/cndb/blob/main/pom.xml#L40) pins that to version 1.4.14.

So CNDB will continue running under 1.4.14. This [link](https://datastax.jira.com/browse/HCD-62?focusedCommentId=724622) is how things stay.

Reply:
All PRs use the same versions now